tasks/util/constants.js

```diff
@@ -133,17 +142,20 @@ var partialBundleTraces = {
         'mesh3d',
         'ohlc',
         'parcats',
+        'parcoords-strict',
         'pie',
         'pointcloud',
         'sankey',
         'scatter',
-        'scatter',
+        'scattergl-strict',
         'scatter3d',
         'scattercarpet',
         'scattergeo',
         'scattermapbox',
         'scatterpolar',
+        'scatterpolargl',
```

Comment:
Even if `scatterpolargl` could become `strict` this way, it is rather confusing. 
Could we add a `scatterpolargl-strict` file instead?

Reply:
Yes, good catch! In case someone only wants polar it would feel wrong to need to include the cartesian one (even if, as I suspect is the case, omitting cartesian wouldn't currently save any bytes)

Reply:
So scatterpolargl is a weird one - it used to be excluded from strict because it required regl but only because it used scattergl.

But in the strict build, if scattergl is safe, then scatterpolargl is safe with no changes whatsoever. So it is just simply restoring it with no change because it suddenly just works in the strict environment.

I think because it's actually the unchanged, it is fine to leave this as is?

Reply:
But `scattergl` is only safe if you bring it in via `scattergl-strict`, right? So if you make a custom bundle that brings in `scatterpolargl` without also including any variant of `scattergl`, you'll get the original, unsafe version of `scattergl` via its internal requires, won't you?

Reply:
To answer the question, yes the safety of scatterpolargl depends on the safety of scattergl trace

Reply:
I have addressed this just now, added scatterpolargl-strict which will work in a custom build by itself too.